src/game/client/components/items.cpp

```diff
@@ -301,19 +289,38 @@ void CItems::RenderLaser(const CLaserData *pCurrent, bool IsPredicted)
 		Alpha = g_Config.m_ClShowOthersAlpha / 100.0f;
 	}
 
+	ColorRGBA OuterColor = color_cast<ColorRGBA>(ColorHSLA(ColorOut));
+	ColorRGBA InnerColor = color_cast<ColorRGBA>(ColorHSLA(ColorIn));
+	OuterColor.a = Alpha;
+	InnerColor.a = Alpha;
+
+	float Ticks;
+	if(Type == LASERTYPE_DOOR)
+	{
+		Ticks = 0.02f * Client()->GameTickSpeed();
```

Comment:
Why so roundabout?

```suggestion
		Ticks = 1.0f;
```

Even if the tick speed would be different, this should still just always be one tick, right?